ONSdigital/eq-survey-runner
# Issue 331 navigator issues

### What is the context of this PR?

Fixes #331 and the problems with navigation
### How to review
1. Check out this branch
2. Ensure all the defects reported in that issue are fixed
### Who worked on the PR

@warrenbailey
